sdk-ts/tests/vault.test.ts

```diff
@@ -1,17 +1,28 @@
 import * as anchor from "@project-serum/anchor";
-import { Connection, Keypair } from "@solana/web3.js";
-import { expect } from "chai";
+import { Connection, Keypair, PublicKey } from "@solana/web3.js";
+import SolanaTokenList from "@solana/spl-token-registry/dist/module/tokens/solana.tokenlist.json";
 import Vault from "../src/vault";
 import { SOL_MINT } from "../src/constants";
+import { bs58 } from "@project-serum/anchor/dist/cjs/utils/bytes";
 
-const mockWallet = new anchor.Wallet(Keypair.generate());
-const mockConnection = new Connection("https://api.mainnet-beta.solana.com");
+const mockWallet = new anchor.Wallet(
```

Comment:
- Shouldn't expose the private key here
- Should generate a new private key + request SOL airdrop

Reply:
transfer all your tokens, and burn this wallet lol
it will exist in the commit history forever